src/lib/hooks/swr-extensions.js

```diff
@@ -46,42 +47,45 @@ export function laggy(useSWRNext) {
 
 /* Wrapper for SWR using 'laggy' data */
 export const useMeasurements = (url, fetcher = defaultFetcher) => {
-  const { data, isLagging } = useSWR(url, {
+  const { data, isLagging, error } = useSWR(url, {
     fetcher: () => fetcher(url),
     use: [laggy],
   });
   return {
-    measurements: data?.data,
+    measurements: data?.measurements,
     pagination: data?.pagination,
-    isLoading: !data,
+    isLoading: !data && !error,
     isLagging: isLagging,
+    error: error,
   };
 };
 
 
 export const useSummarizedData = (url, fetcher = defaultFetcher) => {
```

Comment:
I wonder how much sense it makes to have these separated?
at least the naming of `useSummarizedData` should probably be renamed to something like `useSummary` 
or something similar.